src/gui/guiPasswordChange.cpp

```diff
@@ -285,15 +287,14 @@ bool GUIPasswordChange::getAndroidUIInput()
 	if (!hasAndroidUIInput())
 		return false;
 
-	gui::IGUIElement *e;
+	gui::IGUIElement *e = nullptr;
 	if (m_jni_field_name == "old_password")
 		e = getElementFromId(ID_oldPassword);
 	if (m_jni_field_name == "new_password_1")
```

Comment:
i think we can use else_if too , as you are modifying the block

Reply:
@stujones11 any thoughts on this suggestion?

Reply:
@stujones11 what's your opinion on this?